docs/how-to-guides/upgrades.md

```diff
@@ -0,0 +1,42 @@
+Upgrades are done just by running the `juju refresh` subcommand. Juju, Kubernetes, and Discourse then work together to ensure that one pod is upgraded to the new version and makes any database schema changes before the rest of the pods are upgraded in their turn.
+
+## Upgrading from pod-spec to sidecar
+It is recommended to take a database backup before starting an upgrade from pod-spec to the sidecar version of the discourse-k8s charm.
+
+When upgrading from the pod-spec charm some downtime is to be expected. The existing Deployment will be deleted and a new StatefulSet will be created from scratch. Hence, the charm will be unresponsive while the discourse image resource is downloaded from the Charmhub registry and the first unit is spun up.
+
+The [Nginx Ingress Integrator charm](https://charmhub.io/nginx-ingress-integrator) should now be deployed alongside the Discourse charm and related and configured appropriately to provision the ingress resources as they are not provided by sidecar charms. You can do this as follows:
+```
+# After upgrading to the sidecar version
+juju deploy nginx-ingress-integrator
+juju relate nginx-ingress-integrator discourse-k8s
+# If you require TLS (per below):
+juju config nginx-ingress-integrator tls-secret-name=YOUR_TLS_SECRET_NAME
+# If your cluster has RBAC:
+juju trust nginx-ingress-integrator --scope cluster
+```
+
+Because the pod-spec charm doesn't [store state]( https://discourse.charmhub.io/t/keeping-state-in-juju-controllers-in-operator-framework/3303) in the Juju controller, during the upgrade the charm stored state will be lost, so the database relations will have to be recreated in order to exchange the connection details again. This can be done by running:
```

Comment:
You've got a space after the "(" in the link for store state.

Reply:
Fixed